entrypoint.sh

```diff
@@ -32,6 +38,7 @@ sh -c "git push mirror $branch"
 sleep $POLL_TIMEOUT
 
 pipeline_id=$(curl --header "PRIVATE-TOKEN: $GITLAB_PASSWORD" --silent "https://${GITLAB_HOSTNAME}/api/v4/projects/${GITLAB_PROJECT_ID}/repository/commits/${branch}" | jq '.last_pipeline.id')
+echo "******* Pipeline URL: $*/-/pipelines/${pipeline_id} *******"
```

Comment:
Move this line to the end and remove the `*` from the beginning and end of the line.

Reply:
I would like the URL to be available _before_ the pipeline has finished, so I think this line needs to stay here. Alternatively, I could move it inside the loop so that it's printed out every time the pipeline status is printed if you're concerned that it will disappear after the status is printed out several times.

I'm not married to the asterisks, but I was trying to make the line stand out in what can become a sea of output. In the future, it would be cool if the URL were automatically added as a comment to the PR, but that seems somewhat hard.

Reply:
Whoops, I didn't mean the absolute end. Sorry about that. I meant after line 44 (i.e. after `echo "Working with pipeline id ..."`).

It will be the second to last line before the loop, so I think it should be pretty easy to spot, even without the '*'. However, if you're worried about it, I suppose you could add a few '*' between the intro informational output and the loop output. I just think it looks weird that this is the only line with '*', both before and after.

Reply:
OK that makes more sense.

And I think I agree with you that it should be pretty easy to find even without the asterisks. So they're gone now.